spec/request/preneeds/application_forms_request_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe 'Preneeds Application Form Integration', type: :request do
   include SchemaMatchers
+  before(:each) { Redis.current.flushall }
```

Comment:
Is it absolutely necessary to put this in a before(:each) I wonder? I would think we just need to test it once, and then stub it elsewhere. Perhaps if you moved the caching from the controller to a service object or in the initializer for FormFill, you could test that functionality at the UNIT level, and just stub it here.